frontend/src/components/MilestoneEdit.tsx

```diff
@@ -83,7 +99,13 @@ const MilestoneEdit: FC<MilestoneEditProps> = ({
         <Button disabled>Speichern</Button>
```

Comment:
```suggestion
        <Button disabled={!{formData.title.trim() && formData.dueDate.trim()}}>Speichern</Button>
```
becomes more readable when you trim the data before setting it to the new state. You can always introduce a variable before the return to help with readability